wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -348,16 +255,15 @@ def run(
         if launch_project.docker_image:
             if len(containers) > 1:
                 raise LaunchError(
-                    "Multiple container configurations should be specified in a yaml file supplied via job_spec."
+                    "Multiple container configurations should be specified under kubernetes."
```

Comment:
I think we need better phrasing for "under kubernetes". It's weird, but in the UI, the users don't know they are putting the keys under a kubernetes key. We may just need to link to the docs for how to submit a job via the CLI and via the UI.

Reply:
I think this error could say something like:
"To specify multiple containers, please provide the container images in the job specification"